cmd/minishift/cmd/util/env.go

```diff
@@ -0,0 +1,32 @@
+/*
+Copyright (C) 2017 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"os"
+	"regexp"
+)
+
+const (
+	booleanMatch = "^(?i:true|y|yes|1|on)$"
+)
+
+func GetBoolEnv(varName string) bool {
```

Comment:
At the moment it is explicit for environment variables. For me this lives on the border between cmd and pkg... and since it is only used in cmd, I stuffed it there.